telluric/georaster.py

```diff
@@ -1629,7 +1630,7 @@ def get_window(self, window, bands=None,
             affine = affine or self._calculate_new_affine(window, out_shape[2], out_shape[1])
             raster = self.copy_with(image=array, affine=affine)
 
-            if masked and (raster.image.mask is np.ma.nomask):
+            if masked and not raster.image.mask.any():
```

Comment:
If raster.image.mask is np.ma.nomask, won't this fail? #97

Reply:
No:

```
In [1]: import numpy as np

In [2]: np.ma.nomask.any()
Out[2]: False
```

Reply:
Interesting:

```
>>> ma.nomask is False
False
>>> ma.nomask.any()
False
>>> False.any()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'bool' object has no attribute 'any'
>>> type(ma.nomask)
<class 'numpy.bool_'>
```